zksync_error/errors/evm.json

```diff
@@ -0,0 +1,31 @@
+{
+  "types": [],
+  "domains": [
+    {
+      "domain_name": "OS",
```

Comment:
Not sure what "domain" should be, is it expected to be the same as in zksync-os.json?